plugins/versionlock.py

```diff
@@ -57,6 +58,9 @@ def config(self):
                        and cp.get('main', 'locklist'))
 
     def sack(self):
+        if not self.cli.demands.resolving:
```

Comment:
This will terminate any non-transactional operation for dnf. That means, that dnf repoquery, repolist, info ... will be terminated.